src/components/toolbox/Chart/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { GRAPH_TYPES } from '../../../constants/chartConstants';
+import { optionsByGraphic } from '../../../utils/chartOptions';
+
+const Chart = ({
+  type,
+  data,
+  width,
+  height,
+  options,
+}) => {
+  const Graph = type && GRAPH_TYPES[type];
```

Comment:
💡 You don't need the condition if you apply `PropTypes.oneOf` as suggested below.
```suggestion
  const Graph = GRAPH_TYPES[type];
```